package.json

```diff
@@ -36,9 +36,9 @@
     "got": "^6.3.0"
   },
   "devDependencies": {
-    "ava": "*",
```

Comment:
This is on purpose, same for `XO`. I do this together with @sindresorhus and some others to make sure that we always test our own projects against the latest versions. This way we can detect bugs from those libraries early on.